test/execution_summary_test.py

```diff
@@ -69,16 +69,90 @@ def requires(self):
         self.assertFalse(d['upstream_missing_dependency'])
         self.assertFalse(d['run_by_other_worker'])
         self.assertFalse(d['still_pending_ext'])
-        s = self.summary()
-        self.assertIn('\n* 3 ran successfully:\n    - 3 Bar(num=', s)
-        self.assertIn('\n* 1 present dependencies were encountered:\n    - 1 Bar(num=1)\n', s)
-        self.assertIn('\n* 1 failed:\n'
-                      '    - 1 Bar(num=0)\n'
-                      '* 1 were left pending, among these:\n'
-                      '    * 1 had failed dependencies:\n'
-                      '        - 1 Foo()\n\n'
-                      'This progress looks :( because there were failed tasks', s)
-        self.assertNotIn('\n\n\n', s)
+        summary = self.summary()
+
+        expected = ['',
+                    '===== Luigi Execution Summary =====',
+                    '',
+                    'Scheduled 6 tasks of which:',
+                    '* 1 present dependencies were encountered:',
+                    '    - 1 Bar(num=1)',
+                    '* 3 ran successfully:',
+                    '    - 3 Bar(num=3,2,4)',
+                    '* 1 failed:',
+                    '    - 1 Bar(num=0)',
+                    '* 1 were left pending, among these:',
+                    '    * 1 had failed dependencies:',
+                    '        - 1 Foo()',
+                    '',
+                    'This progress looks :( because there were failed tasks',
+                    '',
+                    '===== Luigi Execution Summary =====',
+                    '']
+        result = summary.split('\n')
+        self.assertEqual(len(result), len(expected))
+        for i, line in enumerate(result):
+            self.assertEqual(line, expected[i])
+
+    @with_config({'core': {'summary-length': '1'}})
```

Comment:
Yea, don't use section `[core]` for this. Maybe use `execution_summary`, and then the cli parameter will be something like`--execution-summary-length 3`. Right?